src/Uno.UI/UI/Xaml/Controls/TreeView/TreeViewViewModel.cs

```diff
@@ -0,0 +1,873 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation.Collections;
+using TreeNodeSelectionState = Windows.UI.Xaml.Controls.TreeViewNode.TreeNodeSelectionState;
+
+namespace Windows.UI.Xaml.Controls
+{
+	internal class TreeViewViewModel
+	{
+		private bool m_isContentMode;
+		private List<TreeViewNode> m_selectedNodes;
+		private List<object> m_selectedItems;
+		private Dictionary<object, TreeViewNode> m_itemToNodeMap;
+
+		private TreeViewNode m_originNode;
+		private TreeViewList m_TreeViewList;
+
+		public TreeViewViewModel()
+		{
+			//var selectedNodes = new List<TreeViewNode>);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)